### DISCUSSION TITLE
Scheduled feature and de-feature of articles does not work.

### DISCUSSION THREAD

**Discussion by hytsch** [joomla/joomla-cms#43595] [Q&A] [answered]
### Steps to reproduce the problem
Create a post and publish it, enter a time X in the "Publish as main post" field.

### Expected result
The article appears as a featured article at time X.

### Actual result
The field "Publish as featured article" remains empty, the automatic featuring function does not seem to work at all. 

### System information (as far as possible)
Joomla 5, PHP 8.3

**Comment by brianteeman** [comment, marked as answer]
It does work.

You have to set the article as Featured AND the time for when it will start to be featured.

You will then see in the list view that it is marked as Featured but Pending

The UI for this is confusing although its the same as used elsewhere
